src/main/java/me/lokka30/microlib/other/VersionUtils.java

```diff
@@ -192,12 +265,31 @@ public static boolean hasMaterial(String materialStr) {
         }
     }
 
+    /**
+     * Checks whether the server has biome.
+     *
+     * @param biome Type to check if exists.
+     * @return If the type is valid with the current server version.
+     * @see Biome
+     * @since 3.1.3
+     */
+    public static boolean hasBiome(String biome) {
```

Comment:
forgot to add final on the function parameter

Reply:
Will be done as soon as possible, currently working on improving JavaDocs.